package.json

```diff
@@ -26,7 +26,7 @@
     "react-dropzone": "14.2.3",
     "react-ga": "3.3.1",
     "react-google-charts": "4.0.0",
-    "react-hyperscript-helpers": "2.0.0",
+    "react-hyperscript-helpers": "^2.0.0",
```

Comment:
Do we need to update `react-hyperscript-helpers`?

Reply:
Just tested it out on my end and the resulting page is the same in both cases. I am not sure why npm updated it for me, but here's something I found about this:
"The main difference between "react-hyperscript-helper 2.0.0" and "^2.0.0" is the specificity of the version being requested. The former will install exactly version 2.0.0, while the latter will install any version that is 2.x.x but not versions that may break compatibility. It's generally a good practice to use version constraints like "^2.0.0" when you want to allow for patch and minor updates without risking major breaking changes."
Do you think it would be a good idea to include it or should I remove those changes for this branch?

Reply:
I think the changes to `package.json` and `package-lock.json` should be reverted, since they're unrelated to the functionality of the ticket. Otherwise, the changes look good to me.

Reply:
sounds good, will do!

Reply:
@fboulnois Done!